packages/integration-tests/src/api/role.ts

```diff
@@ -1,15 +1,21 @@
-import type { CreateRole, Role, Scope, User } from '@logto/schemas';
+import { type CreateRole, type Role, RoleType, type Scope, type User } from '@logto/schemas';
 
 import { generateRoleName } from '#src/utils.js';
 
 import { authedAdminApi } from './api.js';
 
-export const createRole = async (name?: string, description?: string, scopeIds?: string[]) =>
+export const createRole = async (
+  name?: string,
+  description?: string,
+  type?: RoleType,
+  scopeIds?: string[]
```

Comment:
nit: make it a object? it is kind of silly  to provide a scopeId without other arguments by writing `createRole(undefined, undefined, undefined, scopeIds)`